### ISSUE TITLE
"TypeError: exports.en is not a function" in 1.4.7

### ISSUE THREAD

**Issue by elmeister** [wanasit/chrono#334]
Thank you for the great package.
After yesterday's release `1.4.7`, package started to throw an error `TypeError: exports.en is not a function` in `chrono-node/dist/chrono.js:8819:40`
```Javascript
function strictOption() {
  var strictConfig = {
    strict: true
  };
  return exports.mergeOptions([exports.en(strictConfig), exports.de(strictConfig), exports.nl(strictConfig), exports.pt(strictConfig), exports.es(strictConfig), exports.fr(strictConfig), exports.ja(strictConfig), exports.zh, exports.commonPostProcessing]);
}
```

**Comment by wanasit** [comment]
Thanks for letting me know and really sorry for the problem.
I've deprecated `1.4.7` and published `1.4.8` with the fix. Could please update?

**Comment by elmeister** [comment]
Now it works just fine. Thank you for the fast fix.
